beheer.md

```diff
@@ -121,26 +121,36 @@ Beheren publicatie op GEMMA Online | Ja | Zelf
+Vanuit en samen met die community wordt dat beheerteam daarna voor die betreffende standaard verantwoordelijk voor:
 1. beantwoorden van vragen;
 2. het oplossen van problemen;
 3. doorontwikkeling van de standaard.
 
 **Beantwoorden van vragen**
+Vragen mbt. de betreffende standaard kunnen beheer op verschillende manieren bereiken:
+* Mail/telefoon
+* Discussieforum _(NB doen we nog iets met een discussieforum of verlaten we dat pad?)_
+* Slack
+* Issue in Github/Gitlab
+* Pullrequest in Github/Gitlab
 
-Voor het beantwoorden van vragen m.b.t. de betreffende standaard zijn twee kanalen beschikbaar. Ten eerste het specifiek voor de standaard beschikbaar gestelde kanaal in de Slack workspace [VNG API Community](https://vngapicommunity.slack.com) en ten tweede het eveneens specifiek voor de standaard beschikbaar gestelde GitHub/GitLab omgeving.
+Vragen via mail of telefoon hebben niet de voorkeur maar dit gaat ongetwijfeld voorkomen. Bij kleinere vragen hoeft dit geen probleem te zijn maar omwille van de openheid en de herbruikbaarheid van antwoorden kan het nuttig zijn deze alsnog via een discussieforum, Github/Gitlab of Slack te delen. Een andere optie om de vragen en antwoorden langdurig te delen is een veelgestelde vragenlijst samenstellen die in de loop van de tijd groeit. Zo'n lijst moet ergens goed vindbaar gepubliceerd worden, bijvoorbeeld op GEMMA Online of in de repository van de betreffende standaard. 
+
+Voor het beantwoorden van vragen m.b.t. de betreffende standaard zijn meerdere kanalen beschikbaar:
+1. Het specifiek voor de standaard beschikbaar gestelde kanaal in de Slack workspace [VNG API Community](https://vngapicommunity.slack.com) 
+2. Een eveneens specifiek voor (deze versie van) de standaard beschikbaar gestelde GitHub/GitLab omgeving.
+3. Een specifiek discussieforum voor de betreffende standaard.
 
 Slack leent zich niet zo goed voor het gestructureerd voeren van een discussie aangezien in het kanaal meerdere onderwerpen door elkaar heen kunnen gaan lopen en (in de basis variant) historie slechts tot een x aantal reacties beperkt blijft. Slack is vooral geschikt voor het snel stellen en beantwoorden van enkelvoudige vragen. Laagdrempelig contact dus.
 
-Voor het diepgaand bediscusieren van issues en ter sprake stellen van problemen in de standaard waarbij de discussies een lange(re) tijd beschikbaar moeten blijven kan beter gebruik gemaakt worden van GitHub/GitLab.
+Voor het diepgaand bediscusieren van issues en ter sprake stellen van problemen in de standaard waarbij de discussies een lange(re) tijd beschikbaar moeten blijven kan beter gebruik gemaakt worden van GitHub/GitLab. _NB er is nog geen strategie voor repositories en versies van de standaard, waar komen vragen, issues en discussies terecht?_
```

Comment:
Goede vraag. Een opmerking, vraag zal soms een specifieke versie van een standaard betreffen maar soms ook generiek van aard zijn. Voor issues en 'Pull Requests' geldt overigens hetzelfde. Daarom is het denk ik niet handig om, als we daarvoor gaan kiezen, vragen en opmerkingen (en dus ook issues en 'Pull Requests') vast te laten leggen in de publicatie repos. 
Dat betekent dat ze dus in de master repository vastgelegd moeten gaan worden en dat de vraagsteller heel duidelijk moet gaan aangeven om welke versie van de standaard het gaat. De vraag is hoe we dat het handigste kunnen gaan doen.

Reply:
Al kijkende naar het maken van releases met Github denk ik dat we vanuit 1 repository moeten blijven werken.  Zie ook https://help.github.com/en/articles/creating-releases